test-e2e/manager-basic.js

```diff
@@ -56,20 +59,90 @@ test('Managing created projects', async (t) => {
   t.ok(project1)
   t.ok(project2)
 
-  t.test('initial settings from project instances', async (st) => {
-    const settings1 = await project1.$getProjectSettings()
-    const settings2 = await project2.$getProjectSettings()
-
-    st.alike(settings1, {
-      name: undefined,
-      defaultPresets: undefined,
-    })
+  await t.test(
+    'initial settings and default config from project instances',
+    async (st) => {
+      const settings1 = await project1.$getProjectSettings()
+      const settings2 = await project2.$getProjectSettings()
+
+      const {
+        presets: defaultPresets,
+        fields: defaultFields,
+        icons: defaultIcons,
+      } = await getExpectedConfig(
```

Comment:
Naming suggestion: call this `const expected = await getExpectedConfig()`
Then in the asserts below you can do `t.alike(settings1.defaultpresets.point, expected.presets)` etc.